torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -180,36 +180,34 @@ def powerSGD_hook(
     Once gradient tensors are aggregated across all workers, this hook applies
     compression as follows:
 
-    1. Views the input flattened 1D gradient tensor as two groups of per-parameter tensors: high-rank tensors and vector-like rank-1 tensors (for biases).
+    1. Views the input flattened 1D gradient tensor as a list of per-parameter tensors, and divides all the tensors into two groups:
 
-    2. Divides all the tensors into two groups:
+        1.1 The tensors that should be compressed before allreduce, because the compression can give enough saving in bandwidth.
```

Comment:
Do we specify somewhere how we determine which tensors are compressed vs uncompressed (rank 1 & the compression threshold?)

Reply:
It's specified in `_should_compress` method, noted in the precursor PR.